core/src/aaa/main/screens/MainScreen.java

```diff
@@ -0,0 +1,241 @@
+package aaa.main.screens;
+
+import aaa.main.AntGame;
+import aaa.main.stages.PauseMenu;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input;
+import com.badlogic.gdx.ScreenAdapter;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.OrthographicCamera;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.math.MathUtils;
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.math.Vector3;
+import com.badlogic.gdx.physics.box2d.*;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.InputListener;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Image;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+
+import static aaa.main.util.Constants.PPM;
+import static aaa.main.util.Constants.BORDER_WIDTH;
+import static aaa.main.util.Constants.BORDER_HEIGHT;
+import static aaa.main.util.Constants.SCREEN_WIDTH;
+import static aaa.main.util.Constants.SCREEN_HEIGHT;
+
+
+public class MainScreen extends ScreenAdapter {
+    private final AntGame game;
+    private final PauseMenu pauseMenu;
+    private final Stage stage;
+    private boolean pauseOpened = false;
+
+
+    private boolean DEBUG = false;
+    private Box2DDebugRenderer b2dr;
+    private OrthographicCamera camera;
+    private World world;
+    private Body player,borderUP, borderDOWN, borderLEFT, borderRIGHT;
+    private final float SCALE = 2.0f;
+
+
+    public MainScreen(final AntGame game) {
+        this.game = game;
+        stage = new Stage();
+
+
+        //Camera initilization
+        camera = new OrthographicCamera();
+        camera.setToOrtho(false, SCREEN_WIDTH / SCALE , SCREEN_HEIGHT / SCALE);
+
+        //World/debug renderer initialization
+        world = new World(new Vector2(0, 0), false);
+        b2dr = new Box2DDebugRenderer();
```

Comment:
this works as a decent test, but in the future we should likely try to move as much as we can *out* of the screen files, so there are less conflicts

Reply:
not something that needs to be changed now, and this is fine, just something to be aware of in the future